plugins/hammer/package.json

```diff
@@ -15,6 +15,7 @@
   "module": "dist/picasso-hammer.esm.js",
   "scripts": {
     "build": "cross-env NODE_ENV=production rollup --config ../../rollup.config.js",
+    "build:dev": "cross-env rollup --config ../../rollup.config.js",
```

Comment:
no need for `cross-env` unless you're actually setting an environment variable

Reply:
Updated!